implementation/contracts/deposit/DepositFunding.sol

```diff
@@ -60,9 +60,11 @@ library DepositFunding {
         // TODO: Whole value is stored as funder bond in the deposit, but part
         // of it should be transferred to keep: https://github.com/keep-network/tbtc/issues/297
         _d.lotSizeSatoshis = _lotSize;
-        uint256 _bondRequirement = _lotSize.mul(_system.getInitialCollateralizedPercent()).div(100);
+        uint256 _bondRequirementSatoshi = _lotSize.mul(_system.getInitialCollateralizedPercent()).div(100);
```

Comment:
Time to go ahead and change `_lotSize` to specify its unit as well. Let's change it to `_lotSizeSatoshis`, and let's also go through the code and make sure anywhere we talk about denominated numbers we're including the unit in the respective variables.